tests/e2e/lib/kube_helpers.go

```diff
@@ -247,12 +247,43 @@ func GetPodContainerLogs(namespace, podname, container string) (string, error) {
 		return "", err
 	}
 	defer podLogs.Close()
-
 	buf := new(bytes.Buffer)
 	_, err = io.Copy(buf, podLogs)
 	if err != nil {
 		return "", err
 	}
-	str := buf.String()
-	return str, nil
+	return buf.String(), nil
+}
+
+func GetDeploymentPodContainerLogs(namespace, deploymentName, containerName string) (string, error) {
```

Comment:
oh snap.. am I reading the correctly? We have to verify the that restore worked w/ datamover by scraping the deployment log?

Reply:
It's used [here](https://github.com/openshift/oadp-operator/pull/863/files#diff-a9f567f9b6083a8f8e52d9318806042225c729fa99bea2354e148eead33bdf9fR146) in case of failure for dumping out logs